portals/publisher/source/src/app/data/api.js

```diff
@@ -2969,11 +2972,12 @@ class API extends Resource {
     /**
      * Add an API specific operation policy
      * @param {Object} policySpec policy specification of the operation policy
-     * @param {any} policyDefinition policy definition of the operation policy
+     * @param {any} synapsePolicyDefinition policy definition of the operation policy for Synapse
+     * @param {any} ccPolicyDefinition policy definition of the operation policy for Choreo Connect
      * @param {String} apiId UUID of the API
      * @returns {Promise} Promise containing added operation policy specification
      */
-    static addOperationPolicy(policySpec, policyDefinition, apiId) {
+    static addOperationPolicy(policySpec, synapsePolicyDefinition = null, ccPolicyDefinition = null, apiId) {
```

Comment:
usually, we put the optional arguments at the end of args list, so that callee can use the function omitting the optional arguments

Reply:
Fixed with d821039a80da3bb86717401a8b575cd9cf473dfd